model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -538,6 +542,22 @@ async def create_text_generation_inference_bundle(
 
     def load_model_weights_sub_commands(
         self, framework, framework_image_tag, checkpoint_path, final_weights_folder
+    ):
+        if checkpoint_path.startswith("s3://"):
+            return self.load_model_weights_sub_commands_s3(
+                framework, framework_image_tag, checkpoint_path, final_weights_folder
+            )
+        elif checkpoint_path.startswith("azure://") or "blob.core.windows.net" in checkpoint_path:
```

Comment:
hope no one does s3.amazonaws.com/some/path/to/blob.core.windows.net lmao
(I don't think we actually need to worry about this)